source/base/tensor.inst.in

```diff
@@ -0,0 +1,28 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+for (rank : RANKS; deal_II_dimension : DIMENSIONS; number : REAL_SCALARS)
```

Comment:
there is not much use of complex scalars at the moment, but for consistency shouldn't the Tensor class be also instantiated for those?